foundation/www/docs/user/manual/en/human-resources/shift-management.md

```diff
@@ -0,0 +1,47 @@
+<!-- add-breadcrumbs -->
+
+# Shift Management
```

Comment:
Restored docs via: https://github.com/erpnext/foundation/pull/218. The PR bot was inactive for a couple of months, so we had missed updates here. Can you verify and update with the already present files?